demisto_sdk/commands/common/constants.py

```diff
@@ -221,21 +219,344 @@ def no_common_server_python(path):
+        return f"IM103 - {file_path} isn't an image file or unified integration file."
+
+    @staticmethod
+    def no_image_field_in_yml(file_path):
+        return f"IM104 - {file_path} is a yml file but has no image field."
+
+    @staticmethod
+    def image_field_not_in_base64(file_path):
+        return f"IM105 - {file_path}'s image field isn't in base64 encoding."
+
+    @staticmethod
+    def default_image_error(file_path):
+        return f"IM106 - {file_path} is the default image, please change to the integration image."
+
+    @staticmethod
+    def invalid_incident_field_name(word, file_path):
+        return f"IF100 - The word {word} cannot be used as a name, please update the file {file_path}."
+
+    @staticmethod
+    def invalid_incident_field_content_key_value(content_value, file_path):
+        return f"IF101 - The content key must be set to {content_value}, please update the file '{file_path}'"
+
+    @staticmethod
+    def invalid_incident_field_system_key_value(system_value, file_path):
+        return f"IF102 - The system key must be set to {system_value}, please update the file '{file_path}'"
+
+    @staticmethod
+    def invalid_incident_field_type(file_path, file_type, TypeFields):
+        return f"IF103 - {file_path}: type: `{file_type}` is not one of available type.\n" \
+               f"available types: {[value.value for value in TypeFields]}"
+
+    @staticmethod
+    def invalid_incident_field_group_value(file_path, group):
+        return f"IF104 - {file_path}: group {group} is not a group field."
+
+    @staticmethod
+    def invalid_incident_field_cli_name_regex(file_path, cli_regex):
+        return f"IF105 - {file_path}: Field `cliName` contains non-alphanumeric letters. " \
+               f"must match regex: {cli_regex}"
+
+    @staticmethod
+    def invalid_incident_field_cli_name_value(file_path, cli_name):
+        return f"IF106 - {file_path}: cliName field can not be {cli_name} as it's a builtin key."
+
+    @staticmethod
+    def incident_field_or_type_from_version_5(file_path):
+        return f'IF107 - {file_path}: fromVersion must be at least 5.0.0'
+
+    @staticmethod
+    def invalid_incident_field_or_type_from_version(file_path):
+        return f'IF108 - {file_path}: "fromVersion" has an invalid value.'
+
+    @staticmethod
+    def new_incident_field_required(file_path):
+        return f'IF109 - {file_path}: new incident fields can not be required. change to:\nrequired: false.'
+
+    @staticmethod
+    def incident_field_type_change(file_path):
+        return f'IF111 - {file_path}: Changing incident field type is not allowed.'
+
+    @staticmethod
+    def incident_type_integer_field(file_path, field):
+        return f'IF112 - {file_path}: the field {field} needs to be a positive integer. Please add it.\n'
+
+    @staticmethod
+    def invalid_context_output(command_name, output):
+        return f'IN120 - Invalid context output for command {command_name}. Output is {output}'
```

Comment:
CTX

Reply:
there are only two errors connected to context (which are not dbot score nor BC related) - do we want to create a section for those?